website/docs/tutorial/1a-setting-up-bigquery.md

```diff
@@ -0,0 +1,225 @@
+
+    4. Grant users access:
+
+        1. Add your own email initially  
+            <Lightbox src="/img/bigquery/api-grant-user-access.png" title="Bigquery API - Grant Users Access" />
+
+        2. Click “DONE”
+
+3. You now have a service account. Click on it to proceed:
+    <Lightbox src="/img/bigquery/api-select-service-account.png" title="Bigquery API - Select Service Account" />
+
+    1. Under “KEYS” heading, click “ADD KEY” then “Create new key”
+        <Lightbox src="/img/bigquery/api-create-key.png" title="Bigquery API - Create New Key" />
+
+    2. "Key type" `JSON`
+        <Lightbox src="/img/bigquery/api-key-type.png" title="Bigquery API - Key Type" />
+
+    3. Click “CREATE” and it will download the keyfile
+
+## Set up a dbt Cloud project and connect to BigQuery
+
+<LoomVideo id="c75212052038463b977d0eb856d25ff1" />
+
+Above we created a project in Bigquery and created an API key so we interact with BigQuery from other applications. Now we are going to navigate to dbt Cloud and see if we can access our BigQuery Project!
+
+1. It's time to leave BigQuery and navigate to https://cloud.getdbt.com/ so we can create a new project
+
+2. Click the Hamburger
+
+3. Click "Account Settings"
+    
+    1. You should be in the "Projects" section by default. Click "New Project”, in the top right.
+        <Lightbox src="/img/bigquery/set-up-new-project.png" title="dbt Cloud - Set Up A New Project" />
+    
+    2. Click “Begin”
+
+4. Project Settings
+    1. Name your project. I chose 'BigQuery Demo', but you might consider your company's name or 'Arrakis' if you semi-haphazardly name all your projects and repos after Dune references. Click “Continue”.
+        <Lightbox src="/img/bigquery/project-name.png" title="dbt Cloud - Project Name" />
+    
+    2. Set Up a Database Connection
+        <Lightbox src="/img/bigquery/database-connection.png" title="dbt Cloud - Set Up a Database Connection" />
+        1. Click on “BigQuery” icon.
+
+        2. Click on “Upload a Service Account JSON File”.
+        
+        3. Select your file from your recent Downloads folder and dbt Cloud will automatically fill in all the necessary fields:
+            <Lightbox src="/img/bigquery/database-connection-json-file.png" title="dbt Cloud - Upload a JSON File" />
+        
+        4. Click “Test” at the top
+            <Lightbox src="/img/bigquery/database-connection-test.png" title="dbt Cloud - Database Connection Test" />
+        
+        5. If test successful, click “Continue”
+
+    3. Add Repository
+
+        1. If you have one already (CLI folks) you can connect your Github and select it from a dropdown
+
+        2. If you want to use dbt “Managed” repository, you can do that as well.  That's what I am doing, and chose the name 'bigquery_set_up_tutorial'
+            <Lightbox src="/img/bigquery/set-up-repository.png" title="dbt Cloud - Set Up Repository" />
+
+        3. Click “Create” then “Continue”
+
+Welcome to dbt Learn on dbt Cloud!
```

Comment:
quick-fix: I like this call out, but because this is part of the GS tutorial and might be used in different ways, I'd tweak it to something like, "You're ready to jump in on dbt Cloud"

Reply:
Yeah I dont know why it said that to be honest. I actually had a commit to fix that which I forgot to push.  Pushing that now.